packages/fxa-payments-server/src/routes/Product/AcceptedCards/en.ftl

```diff
@@ -0,0 +1,5 @@
+## Routes - Product/AcceptedCards
+## Used in both Routes - Checkout and Product/SubscriptionCreate
+
+pay-with-heading-card-or = Or pay with card
```

Comment:
We can remove this if the id will no longer be used, correct?

Reply:
Correct. Removed. 😄